concent_api/core/templatetags/admin_tags.py

```diff
@@ -1,17 +1,55 @@
-from django import template
+import datetime
 from constance import config
+from django import template
+from django.db.models import ExpressionWrapper
+from django.db.models import F
+from django.db.models import Func
+from django.db.models import IntegerField
+from django.db.models import Q
+from django.db.models import Value
+from django.db.models.functions import Greatest
+from common.helpers import parse_timestamp_to_utc_datetime
+from common.helpers import get_current_utc_timestamp
 from core.models import Subtask
 
 register = template.Library()
 
+active_state_names = [state.name for state in Subtask.ACTIVE_STATES]
```

Comment:
Those are kind of a constants, so upper letters for both.

Reply:
Done.